test/gtest/ucp/test_ucp_tag_offload.cc

```diff
@@ -19,6 +19,8 @@ class test_ucp_tag_offload : public test_ucp_tag {
     void init()
     {
         m_env.push_back(new ucs::scoped_setenv("UCX_TM_ENABLE", "y"));
+        /* TODO: update tag offload tests to operate on 2+ lanes */
+        modify_config("MAX_RNDV_LANES",  "1");
```

Comment:
is this test issue? what is it?

Reply:
yes, issue in the test, @brminich confirmed test issue

Reply:
can u pls comment what is the problem?

Reply:
The test is written for 1 active RX interface, with multi-rail we have several, so receive goes thru another flow (hashing). Due to that, expected behavior slightly differs (counters of SW requests, etc)